policies/spending/travel.md

```diff
@@ -0,0 +1,34 @@
+# Travel spending policy
+
+The Leadership Council of the Rust Project has allocated up to $75,000 USD to be spent over the course of 2024 supporting Members of the Rust Project in attending Rust-related events through the awarding of Travel Grants. These grants are intended to cover travel, hotel, and event tickets for those who otherwise cannot afford to attend. This document sets out the policy for what is allowed for these grants, and the process for the approval of these applications.
+
+Note that this policy is currently experimental. These guidelines may change at any time, as we are currently evaluating the effectiveness of this policy. As we gain experience, we hope to extend this as an ongoing program in the future.
+
+## Approval guidelines
+
+The following are guidelines for approval of an application:
+
+- Up to $2000 per person (per year) (as long as budget remains).
+- You don't have an employer that can pay for you.
+- You are an active member of the Rust project.
+- Your travel costs are kept to a reasonable minimum (i.e. fly economy class, no five star hotels, etc.).
+
+The usual Foundation due diligence on grantees will also be required to ensure compliance with US sanctions, proper record-keeping, anti-fraud measures, etc.
+
+### Automatic approval
+
+In order to reduce the overhead of processing applications, the Foundation will automatically approve applications with the following guidelines:
+
+- The applicant is a Member of the Rust Project (as defined by being on the “All At” mailing list).
+- The event is considered to be legitimate. Whilst there is no definition of a “formal” Rust event, it should be one that is broadly known within the Rust Community and considered to be of value.
+- The amount being sought does not exceed $1,000 USD AND, if awarded, the total amount awarded to that individual including earlier travel grants, should they exist, does not exceed $1,000 USD over the course of a calendar year. The amounts should seem reasonable given the applicant's place of residence and the location of the event itself.
```

Comment:
Maybe contrast this with the $2k number above? e.g. 1k total is the max "auto approve" limit, 2k total is the max "council approve" limit.

Reply:
> 2k total is the max "council approve" limit

I don't think the council really has a limit, right? We could approve larger grants if necessary, on a case-by-case basis.

I think the text here just has a typo. I think it should be:

> The amount being sought does not exceed $1,000 USD AND, if awarded, the total amount awarded to that individual including earlier travel grants, should they exist, does not exceed **$2,000** USD over the course of a calendar year.

Reply:
I'm going by the email you sent, which put a cap at $2000. We can raise the cap if we want, but I think that should be a separate PR that we can make an independent decision on. The intent of this one is to just document our existing policy.

I'm not sure that is a typo, since it was written as $1000 in two different places. My understanding is that the $1000 automatic cap was just intended to be conservative since we were uncertain how fast it would be spent, and we wanted to at least have some council eyes on things. It sounds like a reasonable change to me, and I think we can talk with Paul about making that.